lib/book.rb

```diff
@@ -0,0 +1,12 @@
+class Book
```

Comment:
Three PORO classes can be moved to a nested folder because they're different from rest of the files :)